### ISSUE TITLE
VideoPlayer node reverse video

### ISSUE THREAD

**Issue by sanyi12** [godotengine/godot-proposals#3150]
### Describe the project you are working on

Not relevant

### Describe the problem or limitation you are having in your project

I need to make a forward video and a reverse video.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Just a check box or function with play video in reverse.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

No idea how reverse videos work.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

The workaround is to make another video in reverse, when put in the game folder will increase space.

### Is there a reason why this should be core and not an add-on in the asset library?

It's just a quality of life little checkbox that may be implemented fairly easily.

**Comment by LightningAA** [comment]
See also https://github.com/godotengine/godot-proposals/issues/920

**Comment by AThousandShips** [comment]
Indeed, the simple solution is to use external software to encode your video in reverse, then you will get the compression as well as reverse

As this is extremely specific (most people won't need this) I'd say this is a sufficient workaround

**Comment by berarma** [comment]
This would be very hard to implement and would perform poorly. Video codecs use incremental inter-frame compression that works only in one direction. Frames would have to be first decoded forward and stored to play them reversed later. Video containers are also designed to be read forward and that would make it harder for video and audio too.

The workaround is the best option.

**Comment by Calinou** [comment]
Closing per the above comments, as this is not technically feasible to implement.
